src/autoload/CFG_GLPI.php

```diff
@@ -130,6 +130,14 @@
     'Appliance', 'DatabaseInstance', 'Cable', 'Unmanaged', 'PassiveDCEquipment'
 ];
 
+$CFG_GLPI["rule_types"]                  = ['Computer', 'Monitor', 'NetworkEquipment',
```

Comment:
Maybe a more precise name. We may need to add a similar entry for another rule type in the future.

```suggestion
$CFG_GLPI["ruleimportasset_types"]                  = ['Computer', 'Monitor', 'NetworkEquipment',
```

Reply:
The side effect is that it enable unexpected meta search criteria.

`getMetaParentItemtypesForTypesConfig()` is a bit too permissive. A quick patch could be:
```diff
diff --git a/src/Glpi/Search/SearchEngine.php b/src/Glpi/Search/SearchEngine.php
index aff4a39f70..a594a2e677 100644
--- a/src/Glpi/Search/SearchEngine.php
+++ b/src/Glpi/Search/SearchEngine.php
@@ -193,7 +193,7 @@ final class SearchEngine
         }
 
         $itemclass = $matches[1];
-        if (is_a($itemclass, \CommonDBTM::class, true)) {
+        if (is_a($itemclass, \CommonDBTM::class, true) && !is_a($itemclass, \Rule::class, true)) {
             return [$itemclass::getType()];
         }
```